README.md

```diff
@@ -50,8 +50,23 @@ Your ingress set up then looks like this:
 
 Where the envoy nodes are loadbalancing between each cluster for a given ingress.
 
-### Health Check
-Yggdrasil always configures a path on your Envoy nodes at `/yggdrasil/status`, this can be used to health check your envoy nodes, it will only return 200 if your nodes have started and been configured by Yggdrasil.
```

Comment:
why have you deleted this?

Reply:
My bad, I didn't read the text and assumed it was about the health check annotation (that we used to always configure). Will fix.